Socket.IO-Client-Swift.xcodeproj/project.pbxproj

```diff
@@ -573,9 +570,6 @@
 		572EF2131B51F12F00EEBB58 /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				BITCODE_GENERATION_MODE = bitcode;
-				CODE_SIGN_IDENTITY = "Developer ID Application";
-				ENABLE_BITCODE = YES;
```

Comment:
This isn't defined in the targets, so you'll need to set it to `YES` explicitly for `iOS` and `tvOS`.